spring-web-modules/spring-rest-http-3/src/test/java/com/baeldung/httpstatuscode/StatusControllerUnitTest.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.httpstatuscode;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+@WebMvcTest(StatusController.class)
+public class StatusControllerUnitTest {
```

Comment:
If this is junit5, can we remove access modifiers to the class and test methods?

Reply:
Sure @acerasoni , Thank you